edge-modules/iotedge-diagnostics-dotnet/docker/linux/arm32v7/base/Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG base_tag=3.1.5-bionic-arm32v7
```

Comment:
you don't need to this docker file as we only need 1 copy of docker file for module-base docker image.